cpp/include/raft/distance/detail/kernels/kernel_matrices.cuh

```diff
@@ -353,7 +354,7 @@ class RBFKernel : public GramMatrixBase<math_t> {
     math_t gain   = this->gain;
     using index_t = int64_t;
 
-    auto fin_op = [gain] __device__(math_t d_val, index_t idx) { return exp(-gain * d_val); };
+    rbf_fin_op fin_op{gain};
```

Comment:
This is used ensure that the rbf distance kernel can pre-compiled?

Reply:
Yes. I have now added documentation to `rbf_fin_op.cuh`.